tests/parsers/sqlite_plugins/macos_tcc.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Tests for the macOS TCC plugin."""
+
+from __future__ import unicode_literals
+
+import unittest
+
+from plaso.parsers.sqlite_plugins import macos_tcc
+
+from tests.parsers.sqlite_plugins import test_lib
+
+
+class MacOSTCCPluginTest(test_lib.SQLitePluginTestCase):
+  """Tests for the MacOS TCC plugin."""
+
+  def testProcess(self):
+    """Tests the Process function on a macOS TCC file."""
+    plugin = macos_tcc.MacOSTCCPlugin()
+    storage_writer = self._ParseDatabaseFileWithPlugin(
+        ['TCC-test.db'], plugin)
+
+    self.assertEqual(storage_writer.number_of_warnings, 0)
+    self.assertEqual(storage_writer.number_of_events, 21)
+
+    events = list(storage_writer.GetEvents())
```

Comment:
Please use CheckEventValues also see https://github.com/log2timeline/plaso/blob/master/tests/parsers/winprefetch.py#L70 for an example

Reply:
Done